src/drive/share/index.ts

```diff
@@ -267,28 +267,17 @@ export class Share {
     );
   }
 
-  /**
-   * Get private folder data.
-   *
-   * @param {string} folderUUID - The UUID of the folder.
-   * @returns {Promise<{ data: PrivateSharedFolder }>} A promise containing the private folder data.
-   */
-  public getPrivateSharedFolder(folderUUID: string): Promise<{ data: PrivateSharedFolder }> {
-    return this.client.get(`private-sharing/by-folder-id/${folderUUID}`, this.headers());
-  }
-
```

Comment:
This has been deleted because the endpoint no longer works?

Reply:
Yep, I think so. Now we use sharings/... instead of private-sharing/...